harness/determined/cli/cli.py

```diff
@@ -46,66 +46,52 @@
     version,
     workspace,
 )
-from determined.common import api, util, yaml
+from determined.common import api, util
 from determined.common.api import bindings, certs
 
 
+def _render_search_summary(resp: bindings.v1PreviewHPSearchResponse) -> str:
+    output = [
+        termcolor.colored("Using search configuration:", "green"),
+    ]
+
+    # For mypy
+    assert resp.summary and resp.summary.config and resp.summary.runs
+    # Exclude empty configs from rendering.
+    searcher_config = {k: v for k, v in resp.summary.config.items() if v is not None}
+
+    config_str = render.format_object_as_yaml(searcher_config)
+    output.append(config_str)
+    headers = ["Runs", "Training Time"]
+    run_summaries = []
+    for run_summary in resp.summary.runs:
+        num_runs = run_summary.count
+        run_unit = run_summary.unit
+        if run_unit.maxLength:
+            summary = "maximum length of training code"
```

Comment:
I feel like this phrasing is weird.  Do you like "Training time: to completion" better than "Training time: maximum length of training code"?

I think it's the "training **TIME** = maximum **LENGTH**" that feels like somebody is answering a different question than what they were asked, or something.